post-service/src/main/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+# DataSource properties
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+spring.datasource.url=jdbc:postgresql://posts-db-container:5432/postsdb
+spring.datasource.username=root
+spring.datasource.password=password
```

Comment:
Across the entire application port, username and password should be taken from the environment variables